articles/confidential-computing/confidential-vm-overview.md

```diff
@@ -42,12 +42,18 @@ Confidential OS disk encryption is optional, because this process can lengthen t
 For further integrity and protection, confidential VMs offer [Secure Boot](/windows-hardware/design/device-experiences/oem-secure-boot) by default when confidential OS disk encryption is selected. 
 With Secure Boot, trusted publishers must sign OS boot components (including the boot loader, kernel, and kernel drivers). All compatible confidential VM images support Secure Boot. 
 
+
 ### Encryption pricing differences
 
 Azure confidential VMs use both the OS disk and a small encrypted virtual machine guest state (VMGS) disk of several megabytes. The VMGS disk contains the security state of the VM's components. Some components include the vTPM and UEFI bootloader. The small VMGS disk might incur a monthly storage cost.
 
 From July 2022, encrypted OS disks will incur higher costs. This change is because encrypted OS disks use more space, and compression isn't possible. For more information, see [the pricing guide for managed disks](https://azure.microsoft.com/pricing/details/managed-disks/).
 
+## Attached data disk encryption
+If you are using an Azure confidential VM and attaching extra disks, in order to maximize its security benefits follow the below recommendation:
```

Comment:
below recommendation has lots of implications to customer.
Please document it explicitly.